converged_sound_open_fw_modules_development/modules/aca_module/aca_module.cc

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+#if defined(XTENSA_TOOLSCHAIN) || defined(__XCC__)
+#include <xtensa/tie/xt_hifi2.h>
+#endif
+
+#include "aca_module.h"
+
+#include <logger.h>
+
+using namespace intel_adsp;
+
+DECLARE_LOADABLE_MODULE(AcaModule,
+                        AcaModuleFactory)
+
+
+ErrorCode::Type AcaModuleFactory::Create(
+    SystemAgentInterface &system_agent,
+    ModulePlaceholder *module_placeholder,
+    ModuleInitialSettings initial_settings
```

Comment:
I'm assuming these dependencies are in the previous PR ?

Reply:
Yes